src/v0/destinations/mp/transform.js

```diff
@@ -259,11 +259,16 @@ const processAliasEvents = (message, type, destination) => {
       'Either previous id or anonymous id should be present in alias payload',
     );
   }
```

Comment:
const aliasId = message.previousId || message.anonymousId;
 if (aliasId === message.userId)

Reply:
Error message can be `One of previousId/AnonymousId is same as userId`